modules/dmrpp_module/CurlHandlePool.cc

```diff
@@ -384,7 +394,7 @@ CurlHandlePool::get_easy_handle(Chunk *chunk) {
  *
  * @param handle
```

Comment:
We should document 'replace' here. 

I have questions about what this is doing, so I think I need to checkout the branch and look at all the code. The diffs don't show enough.